__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest')
+const app = require('../app')
+const seed = require('../db/seeds/seed')
+const db = require('../db/connection')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+describe('GET /api/topics', () => {
+    it('returns a 200 status code and returns an array of topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((res) => {
+            console.log(res.body.topics)
+            const topics = res.body.topics
+            expect(Array.isArray(topics)).toBe(true)
```

Comment:
We should also test that the length of the array is what we expect it to be - it's a really good way to know we're getting all the data back we expect to get back!